src/scenes/PlantsDashboardRouter/components/NumberOfSpeciesPlantedCard.tsx

```diff
@@ -31,6 +38,74 @@ export default function NumberOfSpeciesPlantedCard({
     return <SiteWithZonesCard newVersion={newVersion} />;
   }
 }
+const RolledUpCard = ({ projectId }: { projectId?: number }): JSX.Element => {
+  const { reportedPlants } = useProjectPlantings(projectId);
+  const { species: orgSpecies } = useSpeciesData();
+
+  const projectTotalSpecies = useMemo(() => {
+    const allSpeciesIds = new Set();
+    for (const site of reportedPlants) {
+      for (const species of site.species) {
+        allSpeciesIds.add(species.id);
+      }
+    }
+    return allSpeciesIds.size;
+  }, [reportedPlants]);
+
+  const labels = [strings.RARE, strings.ENDANGERED, strings.OTHER];
+
+  const speciesMap = new Map();
+
+  const projectSpecies = useMemo(() => {
+    for (const site of reportedPlants) {
+      for (const species of site.species) {
+        if (!speciesMap.has(species.id)) {
+          speciesMap.set(species.id, species);
+        }
+      }
+    }
+
+    return Array.from(speciesMap.values());
+  }, [reportedPlants]);
+
+  const values = useMemo(() => {
+    if (projectSpecies && orgSpecies) {
+      const speciesByCategory: Record<string, number> = {
+        [strings.RARE]: 0,
+        [strings.ENDANGERED]: 0,
+        [strings.OTHER]: 0,
+      };
+      projectSpecies.forEach((reportedSpecies) => {
+        const species = orgSpecies.find((s) => s.id === reportedSpecies.id);
+        if (species) {
+          let endangered = false;
+          let rare = false;
+          if (species.conservationCategory === 'EN' || species.conservationCategory === 'CR') {
+            endangered = true;
+          }
+          if (species.rare) {
+            rare = true;
+          }
+          speciesByCategory[strings.RARE] += rare ? 1 : 0;
+          speciesByCategory[strings.ENDANGERED] += endangered ? 1 : 0;
+          speciesByCategory[strings.OTHER] += !(rare || endangered) ? 1 : 0;
```

Comment:
You don't have to change this, but another way to do this is `Number(rare)`

Reply:
I will leave it this way, since it is done like this in the other charts, but good to know!